plugins/modules/helm.py

```diff
@@ -471,13 +487,13 @@ def main():
 
         if release_status is None:  # Not installed
             helm_cmd = deploy(helm_cmd, release_name, release_values, chart_ref, wait, wait_timeout,
-                              disable_hook, False, atomic=atomic)
+                              disable_hook, False, values_files, atomic=atomic)
             changed = True
 
         elif force or release_values != release_status['values'] \
```

Comment:
You need to update the release_values variable otherwise idempotency will be broke
We looking to helm-diff in #128 to have a better management of idempotency